.gitignore

```diff
@@ -3,5 +3,5 @@
 *.tfstate
 *.tfstate.backup
 plan
-
+.zsh_config
```

Comment:
aside: Is this for directory-level configuration loading/unloading? What tool do you use?

Reply:
It was a one off I tested for something that didn't work. I'll remove it :/